manifests/init.pp

```diff
@@ -36,6 +36,8 @@
 # $enable_ostree::        Boolean to enable ostree plugin. This requires existence of an ostree install.
 #                         type:boolean
 #
+# $github_username::      Github username to add remotes for
+#
```

Comment:
I see the benefit of thought, but as you point out the dash vs hyphen downside. That will also require people to think and know what plugins they want as opposed to just seeing options for them. Which could be OK, I just tend towards the obvious but am happy to be steered towards the flexible.